src/medCore/gui/commonWidgets/medSlider.cpp

```diff
@@ -15,25 +15,35 @@
 
 void medSlider::addTick(int position)
 {
-    if(!ticksList.contains(position))
+    if( (tickCount() == 0) || !ticksList.contains(position))
     {
         ticksList.append(position);
     }
 }
 
 int medSlider::tickCount()
 {
-   return ticksList.count();
+    if (!ticksList.toVector().isEmpty())
```

Comment:
In fact, i do not know at all, but `QList::isEmpty()` crashed my soft when the list was empty. This was the only thing which work. Really weird